layouts/auto_generated_product.tpl

```diff
@@ -1,10 +1,10 @@
 <!DOCTYPE html>
-{% include "template-variables" layout_product: true %}
+{% assign product_page = true %}
```

Comment:
Here and in other places – always use trim included components/assignments when used before `<html` tag to avoid extra empty lines in HTML output.

`{%- -%}`